examples/sites/src/views/components/components.vue

```
@@ -1,763 +1,443 @@
+<script setup lang="ts">
+import { reactive, computed, watch, onMounted, nextTick, ref } from 'vue'
 import { useRoute } from 'vue-router'
-import { marked } from 'marked'
-import hljs from 'highlight.js'
-import { Anchor, ButtonGroup, Grid, GridColumn, Tabs, TabItem, Tooltip } from '@opentiny/vue'
-import { iconOuterLink } from '@opentiny/vue-icon'
+import { TinyButtonGroup, TinyTabs, TinyTabItem } from '@opentiny/vue'
 import { debounce } from '@opentiny/utils'
-import { i18nByKey, getWord, $clone, fetchDemosFile, useApiMode, useTemplateMode, getCmpContributors } from '@/tools'
-import demo from '@/views/components/demo'
+import { i18nByKey, getWord, $clone, fetchDemosFile, useApiMode, useTemplateMode } from '@/tools'
```

ESLint

[error] 'fetchDemosFile' is defined but never used.

(unused-imports/no-unused-imports)